src/constants/addresses.ts

```diff
@@ -54,7 +54,12 @@ export const ADDRESSES: AddressMapping = {
   [AddressName.GNOSIS]: { [ChainId.Bsc]: '0xdbB0DfcB3601e15541c072B2a866C0D53D6c6627' },
   [AddressName.USDC]: { [ChainId.Bsc]: '0x8ac76a51cc950d9822d68b83fe1ad97b32cd580d' },
   [AddressName.ONE_DODO]: { [ChainId.Bsc]: '0x2c30b4cb4b3001afa5b8c43c5a7ca548067562a0' },
-  [AddressName.DLP]: { [ChainId.Bsc]: '0x018e41228b1ebc2f81897150877edbb682272c64' }
+  [AddressName.DLP]: { [ChainId.Bsc]: '0x018e41228b1ebc2f81897150877edbb682272c64' },
```

Comment:
What does DLP stand for?

Reply:
This is not my code - I am not sure what DLP is